src/components/FilePreview/__tests__/FilePreview.test.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+
+import {act, render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import {FilePreview} from '../FilePreview';
+import {CircleExclamation} from '@gravity-ui/icons';
+
+describe('FilePreview', () => {
```

Comment:
I suggest to not to add wrappers for tests, this is already covered by Jest itself